.github/workflows/ruby_test.yml

```diff
@@ -209,9 +210,4 @@ jobs:
       uses: paambaati/codeclimate-action@v2.7.5
       with:
         coverageLocations: |
-          tmp/codeclimate-rake_tests-
-          tmp/codeclimate-rspec_tests-0
-          tmp/codeclimate-rspec_tests-1
-          tmp/codeclimate-rspec_tests-2
-          tmp/codeclimate-cucumber_tests-0
-          tmp/codeclimate-cucumber_tests-1
+          ${{github.workspace}}/tmp/codeclimate-*/coverage.json:simplecov
```

Comment:
Was looking in the wrong location. Plus, now with globbing, so we're not coupled to the steps above.